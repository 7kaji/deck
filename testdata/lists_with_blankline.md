# Lists with blank line

---

# Title

- First list item

- Second list item
