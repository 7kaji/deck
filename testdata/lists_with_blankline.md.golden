[
  {
    "layout": "",
    "titles": [
      "Lists with blank line"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Title"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "First list item"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Second list item"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]